ui/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -372,8 +372,6 @@ function TransactionListItemInner({
           showSpeedUp={shouldShowSpeedUp}
           isEarliestNonce={isEarliestNonce}
           onCancel={cancelTransaction}
-          showCancel={isPending && !hasCancelled && !isBridgeTx}
-          showErrorBanner={Boolean(error)}
```

Comment:
this looks like the banner message:

"This transaction would have cost you extra fees, so we stopped it. Your money is still in your wallet." for all transaction message errors detected. This may have been misleading.